### PR TITLE
UI: Automatically refresh page on logout

### PR DESCRIPTION
**Pull request by hashishaw**
When a Vault user logs out, automatically refresh the page by navigating to it with `location.assign` rather than using the router's `replaceWith` method.

### CODE REVIEW

**Review by Monkeychip — APPROVED** [review]
nicely done. Thank you for tackling!

File: ui/app/routes/vault/cluster/logout.js [hunk 1]
```diff
@@ -16,14 +16,15 @@ export default Route.extend(ModelBoundaryRoute, {
...
+    const authType = this.auth.getAuthType();
+    const baseUrl = window.location.origin;
     this.auth.deleteCurrentToken();
     this.controlGroup.deleteTokens();
     this.namespaceService.reset();
     this.console.set('isOpen', false);
     this.console.clearLog(true);
     this.flashMessages.clearMessages();
     this.permissions.reset();
-    this.replaceWith('vault.cluster.auth', { queryParams: { with: authType } });
```
Comment (meirish) [inline ui/app/routes/vault/cluster/logout.js:27]: Just an fyi that you can use the router service to get a full url by name (but I don't see the auth route changing anytime soon 😂): 
https://api.emberjs.com/ember/3.27/classes/RouterService/methods/urlFor?anchor=urlFor

Comment (hashishaw) [reply ui/app/routes/vault/cluster/logout.js:27]: Thanks for the tip! You never know 😅

Comment (hashishaw) [reply ui/app/routes/vault/cluster/logout.js:27]: For some reason when I import the router service and try to get the path from it, it breaks some test page selectors 🤔 Spent a whole day trying to figure out what was going on with no luck, so going with the hardcoded auth link for now 😞  @meirish if you have any ideas of what might be going on let me know, but this should work for now

**Review by Monkeychip — APPROVED** [review]
checked out the branch and did some testing. looks good!

**Pull request merged**